### PR TITLE
remove the local should and change to a better way

### PR DESCRIPTION
**Pull request by iyuq**
Since I don't like the local should either, I think this is better to use this to avoid local should. It can also pass test on Node v6.0.0. If this make sence, I will also open another PR to the v2.x branch.

### CODE REVIEW

File: test/request/querystring.js [hunk 1]
```diff
@@ -33,8 +32,9 @@ describe('ctx.querystring=', function(){
     ctx.querystring = 'page=2&color=blue';
     ctx.url.should.equal('/store/shoes?page=2&color=blue');
     ctx.search.should.equal('?page=2&color=blue');
-    should(ctx.query).have.property('page', '2');
-    should(ctx.query).have.property('color', 'blue');
+    Object.keys(ctx.query).should.eql(['page', 'color']);
```
Comment (PlasmaPower) [inline test/request/querystring.js]: Why this assert? The next ones will throw an error if the keys don't exist anyway.

**Comment by iyuq** [comment]
unless the keys is set enumerable:false, it will be in the Object.keys. But I'm not sure the order of the two keys, it seems V8 order them in the order they define. I check this to ensure the query object doesn't contain any other key. @PlasmaPower

**Comment by jonathanong** [comment]
i don't see the benefit of moving from a local `should` to a global `should`. i'd actually prefer if we only used local `should`. even more, i'd prefer if we ditched `should` in favor of node.js' built-in `assert`.

anyone else have opinions?

**Comment by jonathanong** [comment]
also, this should be against `v2.x` as that will be maintained longer

**Comment by tj** [comment]
node's assert is brutal, that's my 0.02 haha, I like tests you can actually read, but most importantly assert's error messages are really poor

**Comment by jonathanong** [comment]
only going to merge bug fixes to v1.x now. thanks!

**Pull request closed without merging**